packages/spector/src/actions/server-test.ts

```diff
@@ -194,42 +142,61 @@ export async function serverTest(scenariosPath: string, options: ServerTestOptio
       const obj: ServerTestsGenerator = new ServerTestsGenerator(name, api, baseUrl);
       try {
         await obj.executeScenario();
-        success_diagnostics.push({
-          scenario_name: name,
-          status: "success",
-          message: "executed successfully",
+        successfullScenarios.push({
+          name,
+          pathlikeName,
         });
       } catch (e: any) {
-        failure_diagnostics.push({
-          scenario_name: name,
-          status: "failure",
-          message: `code = ${e.code} \n message = ${e.message} \n name = ${e.name} \n stack = ${e.stack} \n status = ${e.status}`,
-        });
+        if (e instanceof ValidationError) {
+          failureDiagnostics.push({
+            scenarioName: name,
+            message: [
+              `Validation failed: ${e.message}:`,
+              ` Expected:\n    ${inspect(e.expected)}`,
+              ` Actual:\n    ${inspect(e.actual)}`,
+            ].join("\n"),
+          });
+        } else {
+          failureDiagnostics.push({
+            scenarioName: name,
+            message: `code = ${e.code} \n message = ${e.message} \n name = ${e.name} \n stack = ${e.stack} \n status = ${e.status}`,
+          });
+        }
       }
     }
   }
 
   // 4. Print diagnostics
-  logger.info("Server Tests Diagnostics Summary");
+  log("");
+  log("Server Tests Diagnostics Summary");
 
-  // Exit with error if no diagnostics were ran
-  if (success_diagnostics.length === 0 && failure_diagnostics.length === 0) {
```

Comment:
Is it still possible to exit with code `1` if no scenarios were ran? Makes it convenient to detect when an incorrect filter is passed when writing tests.

Reply:
still doing it, just renamed the vars here

tested and it works
<img width="578" alt="image" src="https://github.com/user-attachments/assets/c0f340e8-bdd2-4768-b656-f34e8477e367" />

Reply:
Oh weird - maybe because I was using the vscode reviewer it showed the whole block deleted - nevermind!